src/components/pages/NotFound/PageNotFound.jsx

```diff
@@ -1,27 +1,26 @@
 import React from "react";
 import { Helmet } from "react-helmet-async";
 import "./PageNotFound.scss";
-import Logo from "../../../utils/rct-logo.png";
 import { useNavigate } from "react-router-dom";
+import SearchOffIcon from '@mui/icons-material/SearchOff';
+
 const PageNotFound = () => {
   const navigate = useNavigate();
   return (
     <>
       <Helmet>
         <title>404 Page not found</title>
       </Helmet>
-      <div className="header">
-        <img src={Logo} className="logoImage" alt="Website logo" />
-      </div>
       <div className="pageNotFoundContent">
-        <h1>404 - Page not found.</h1>
-        <h2>Unfortunately, this page doesn't exist.</h2>
+        <SearchOffIcon className='notFoundIcon'/>
+        <h1>404 - Not Found</h1>
+        <h1>Se pare că pagina nu a fost găsită.</h1>
         <h2
           onClick={() => {
             navigate("/home");
           }}
         >
-          Go back home
+          Mergi pe pagina "Acasă"
```

Comment:
Maybe, instead of this line of text, can we add a button with something like "Pagina principala"?